src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,21 +33,29 @@
    -->
    <servlet>
         <servlet-name>hello_servlet</servlet-name>
-	<servlet-class>HelloServlet</servlet-class>
+	    <servlet-class>HelloServlet</servlet-class>
```

Comment:
Indentation in line 36, 41 and 46 differ from the one used in lines 51 and 56. Please choose either one (I'd suggest using indentation like the one used in line 51) and keep it the same for the rest of the code to make reading it simpler and faster.